gensim/test/test_nmf.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 Timofey Yefimov <anotherbugmaster@gmail.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+Automated tests for checking transformation algorithms (the models package).
+"""
+
+import logging
+import unittest
+import numbers
+
+import six
+import numpy as np
+
+from gensim.corpora import mmcorpus, Dictionary
+from gensim.models import nmf
+from gensim import matutils, utils
+from gensim.test import basetmtests
+from gensim.test.utils import datapath, get_tmpfile, common_texts
+
+dictionary = Dictionary(common_texts)
+corpus = [dictionary.doc2bow(text) for text in common_texts]
+
+
+def testRandomState():
+    testcases = [np.random.seed(0), None, np.random.RandomState(0), 0]
+    for testcase in testcases:
+        assert (isinstance(utils.get_random_state(testcase), np.random.RandomState))
```

Comment:
You also need to check that 2 random states with same seed produce the same numbers (simple sanity check)